src/index.html

```diff
@@ -1,12 +1,30 @@
 <!doctype html>
 
 <html lang="en" ng-app="paperHub" ng-strict-di>
-  <head>
+  <head ng-controller="MetaCtrl">
+    <title data-ng-bind="metaService.title"></title>
     <meta charset="utf-8">
-    <title>PaperHub</title>
+    <meta
+      ng-if="metaService.author"
+      name="author"
+      content="{{metaService.author}}"
+      >
+    <meta
+      ng-if="metaService.description"
+      name="description"
+      content="{{metaService.description}}"
+      >
+    <meta
+      ng-if="metaService.keywords"
+      name="keywords"
+      content="{{metaService.keywords}}"
+      >
```

Comment:
I was worried about escaping issues here but both `{{}}` and `ng-bind` take care of HTML-encoding the given expression. So everything's fine. :)